README.md

```diff
@@ -427,6 +427,173 @@ way around. So
+to learn more about how dev archetypes are easily managed with
+`builder-support gen-dev`.
+
+#### Workflow for moving dependencies and scripts to your new archetype
+
+Once everything is configured and `npm link`'d, it should be easy to move
+scripts to your archetype and quickly test them out from a consuming project.
+
+##### Moving `dependencies` and `devDependencies` from an existing `package.json`
+
+* copy `dependencies` to `package.json` `dependencies`.
+* copy `devDependencies` to `dev/package.json` `dependencies`.
+
+##### Moving scripts and config files
+
+All scripts defined in archetypes will be run from the root of the project
+consuming the archetype so all paths in scripts pointing to files also in the
+archetype need to be changed to be referenced at their location within the
+archetype.
+
+An example script and config you may be moving to an archetype would look like:
+
+```js
+"test-server-unit": "mocha --opts test/server/mocha.opts test/server/spec"
+```
+
+When moving this script to an archetype, we'd also move the config from
+`test/server/mocha.opts` within the original project to within the
+archetype such as `config/mocha/server/mocha.opts`.
+
+For this example script, we'd need to update the path to `mocha.opts` as so:
+
+```js
+"test-server-unit": "mocha --opts node_modules/new-archetype-name/config/mocha/server/mocha.opts test/server/spec"
+```
+
+Any paths that reference files expected in the consuming app (in this example
+`test/server/spec`) do not need to change.
+
+##### Updating path and module references within your script configs
+
+Any JavaScript files run from within an archetype (such as config files) require
+a few changes related to paths now that the files are being run from within
+an npm module. This includes all `require()` calls referencing npm modules and
+all paths to files that aren't relative.
+
+For example, `karma.conf.js`:
+
+```js
+module.exports = function (config) {
+  require("./karma.conf.dev")(config);
+
+  config.set({
+    preprocessors: {
+      "test/client/main.js": ["webpack"]
+    },
+    files: [
+      "sinon/pkg/sinon",
+      "test/client/main.js"
+    ],
+  });
+};
+```
+
+All non-relative paths to files and npm modules need to be full paths, even ones
+not in the archetype directory. For files expected to be in the consuming
+project, this can be achieved by prepending `process.cwd()` to all paths. For
+npm modules, full paths can be achieved by using
+[`require.resolve()`](https://nodejs.org/api/globals.html#globals_require_resolve).
+
+An updated config might look like:
+
+```js
+var path = require("path");
+var ROOT = process.cwd();
+var MAIN_PATH = path.join(ROOT, "test/client/main.js");
+
+module.exports = function (config) {
+  require("./karma.conf.dev")(config);
+
+  config.set({
+    preprocessors: {
+      [MAIN_PATH]: ["webpack"]
+    },
+    files: [
+      require.resolve("sinon/pkg/sinon"),
+      MAIN_PATH
+    ],
+  });
+};
+```
+
```

Comment:
As a closing statement, it would be good to provide an example archetype tree.
Here's `builder-victory-component`:

```
.
├── CONTRIBUTING.md
├── HISTORY.md
├── LICENSE.txt
├── README.md
├── config
│   ├── babel
│   ├── eslint
│   ├── karma
│   │   ├── karma.conf.coverage.js
│   │   ├── karma.conf.dev.js
│   │   └── karma.conf.js
│   └── webpack
│       ├── demo
│       │   ├── webpack.config.dev.js
│       │   └── webpack.config.hot.js
│       ├── docs
│       │   ├── webpack.config.dev.js
│       │   ├── webpack.config.hot.js
│       │   └── webpack.config.static.js
│       ├── webpack.config.coverage.js
│       ├── webpack.config.dev.js
│       ├── webpack.config.js
│       └── webpack.config.test.js
├── dev
│   └── package.json
├── package.json
└── util
    └── dev.js
```